### DISCUSSION TITLE
Isaac Gym Baselines with rl_games configurations implemented in SKRL

### DISCUSSION THREAD

**Discussion by noshaba** [Toni-SM/skrl#8] [Ideas]
Hi!

As the title suggests it would be super helpful to have at least one example of the runs and configurations of the Isaac Gym environments currently implemented with the rl_games library implemented in skrl. It doesn't need to be the whole thing, just the PPO, etc. baseline configurations, and which variable maps to which from rl_games to skrl, and having a tutorial about it on your website :)

Best,
Noshaba

**Comment by Toni-SM** [comment]
I will work on it :)

**Comment by Toni-SM** [comment]
Hi @noshaba 

I added, in `develop` branch, examples for all Isaac Gym environments... Only the `timesteps` parameter needs to be adjusted for some environments (I am running all the original Isaac Gym examples to get this value). 

You can see them, for now, in the skrl docs (`develop` branch): https://skrl.readthedocs.io/en/develop/intro/examples.html#learning-in-an-isaac-gym-environment-one-agent-multiple-environments

**Comment by noshaba** [comment]
Thank you so much!! Will check it out!! ^.^

**Reply by Toni-SM** [reply]
Now all parameters are set... in the skrl docs (develop branch)
